src/frontends/common/include/openvino/frontend/decoder.hpp

```diff
@@ -82,6 +82,9 @@ class FRONTEND_API DecoderBase : public IDecoder {
 
     /// \brief Get node name
     virtual const std::string& get_op_name() const = 0;
+
+    /// \brief Destructor
+    virtual ~DecoderBase();
```

Comment:
Is this dtor required?
The public IDector dtor should be sufficient, or there is still some issue without it?

Reply:
it's required because w/o it `DecoderBase` will be header only class and C++ RTTI will not work here

So, moving dtor to .cpp allows to have RTTI info for this class in libopenvino.so

Reply:
In case if such class has some other member already defined in cpp (not pure hpp class) then it should be fine without defined this dtor?

Reply:
I suppose yes